old_api/__init__.py

```diff
@@ -0,0 +1,66 @@
+import hashlib
+from datetime import datetime, timezone
+
+import requests
+from django.conf import settings
+
+from old_api.schema import FileList, FileMetadata
+
+
+session = requests.Session()
+session.headers["Authorization"] = settings.AIRLOCK_API_TOKEN
+
+
+def create_filelist(request):
```

Comment:
This is maybe worth a comment or a type annotation just because `request` is such an overloaded term here.

Reply:
Yep, I'll call it release_request